python/ray/tests/test_placement_group.py

```diff
@@ -280,8 +287,12 @@ def test_remove_pending_placement_group(ray_start_cluster):
     cluster.add_node(num_cpus=4)
     ray.init(address=cluster.address)
     # Create a placement group that cannot be scheduled now.
-    pid = ray.experimental.placement_group([{"GPU": 2}, {"CPU": 2}])
-    ray.experimental.remove_placement_group(pid)
+    placement_group = ray.experimental.placement_group([{
+        "GPU": 2
+    }, {
+        "CPU": 2
+    }])
+    ray.experimental.remove_placement_group(placement_group.id)
```

Comment:
Please change the signature of remove / table API to receive placement group object, not id

Reply:
fixed